pytest_postgresql/executor.py

```
@@ -216,20 +290,86 @@ def running(self) -> bool:
+                    )
+        except (OSError, AttributeError) as e:
+            # Process might already be dead or other issues
+            logger.debug(
+                "Exception during Windows process termination: %s: %s",
+                type(e).__name__,
+                e,
+            )
 
     def stop(self: T, sig: Optional[int] = None, exp_sig: Optional[int] = None) -> T:
         """Issue a stop request to executable."""
         subprocess.check_output(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)